### ISSUE TITLE
Secondary literature table rename: migration tasks

### ISSUE THREAD

**Issue by fjorba** [rism-ch/muscat#1028]
There are a few db migration tasks for the secondary literature table rename.  As they are scattered among different issues, I think that it is better to document them in a single one, so @xhero can find it easily:

Table "catalogues" (rename to: "publications", see #975)

Columns:

t.string "name"  (_rename to_: "short_name", see #1027)
               (_new integer field_ -> "record_type", see #1013)
t.string "author"
               (_new string field_ -> "author_d", to match sources table)
t.string "description" (_rename to_: "title", see #626)
               (_new string field_ -> "title_d", to match sources table)
t.string "revue_title" (_rename to_: "journal", see #626)
               (_new string field_ -> "journal_d", to match sources table)
t.string "volume"
               (_new string field_) -> "number" (not discussed yet anywhere, but it is needed)
t.string "place"
t.string "date"
t.string "pages"
t.integer "wf_audit", default: 0
t.integer "wf_stage", default: 0
t.string "wf_notes"
t.integer "wf_owner", default: 0
t.integer "wf_version", default: 0
t.integer "src_count", default: 0
t.datetime "created_at"
t.datetime "updated_at"
t.text "marc_source"     (change type to: mediumtext, see #792)
t.integer "lock_version", default: 0, null: false
t.index ["name"], name: "index_catalogues_on_name"
t.index ["wf_stage"], name: "index_catalogues_on_wf_stage"

**Comment by xhero** [comment]
I did the first step! The others (like renaming) will need a bit of testing so I would like to do them as a next step. We are planning a release next week, I think we can add all these mods just after (we plan another one in a month or so)

**Comment by fjorba** [comment]
Of course; please let me know which steps am I supposed to do, because I cannot easily distinguish them.  I think that db migration changes should be done by you, @xhero, but if you consider that there is something  in the software that I should take care of, please let me know and I'll do my best.
